cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -139,24 +145,24 @@ export class PassportAuth implements EndpointSecurity {
    *
    * @param passportApi - passport.js instance
    */
-  protected setup(passportApi: passport.Passport) {
-    const options = {
-      jwtFromRequest: ExtractJwt.fromAuthHeader(),
-      secretOrKey: 'secret'
-    };
-    passportApi.use(new Strategy(defaultStrategy(this.userRepo, this.userService)));
-    passportApi.use(new JwtStrategy(options, (jwtPayload, done) => {
-      const callback = (err?: Error, user?: any) => {
-        if (err) {
-          return done(err, false);
-        } else if (user) {
-          return done(null, user);
-        } else {
-          return done(null, false);
-        }
-      };
-      this.userRepo.getUserByLogin(jwtPayload.username, callback);
-    }));
+  protected setup(passportApi: passport.Passport, jwtOpts?: JwtOptions) {
```

Comment:
[Question] Do we still using interface for this? I think we may actually not need 
We can have two setup methods once with jwtOptions and second that will setup session based strategy? This will save you another if and will make code cleaner.
We can even add name to provide more context.

setupToken and setupCookie etc.